test/jdk/java/awt/color/ICC_ProfileSetNullDataTest.java

```diff
@@ -30,22 +30,23 @@
  * @summary Test checks behavior of the ICC_Profile.setData(int, byte[])
  */
 public final class ICC_ProfileSetNullDataTest {
+    private static final int[] colorSpace = new int [] {
```

Comment:
```suggestion
    private static final int[] colorSpace = new int[] {
```

Reply:
In fact, the syntax can be simplified further:
```java
private static final int[] colorSpace = {
```

There's no need for explicit `new int[]`.